hack/lib/golang.sh

```diff
@@ -468,7 +468,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(go version)"
   local minimum_go_version
-  minimum_go_version=go1.13.4
+  minimum_go_version=go1.14.4
```

Comment:
Should this always be the latest go version we've bumped to?
If so, can you add an entry in build/dependencies.yaml for this path/regex?

Reply:
it should, BUT that would require the image bump to go in before the PR, FWIW

Reply:
Ooof. Chicken and egg! Thanks for clarifying, @BenTheElder!
/hold cancel

Reply:
we still have this chicken and egg either way, the question is just which repo we want to merge first :man_shrugging: